components/filter-components/MapFilters.vue

```diff
@@ -154,28 +155,52 @@
 
               <!-- Property titles -->
               <div class="flex pl-2 py-1 flex-wrap items-center mt-2
-                          cursor-pointer" style="width: 225px;" 
+                          cursor-pointer w-full" 
                   @click="togglePropertyShowing(property)">
                 <div class="title-point"></div>
-                <p style="font-weight: 500;"> {{ propertyItem.label }} </p>
+                <!--Descriptions-->
+                <div v-if="!isMobileDevice" class="tooltip">
+                <span class="label-and-icon">
+                  <p style="font-weight: 500; display: inline;"> {{ propertyItem.label }} </p>
+                  <div class="inline-flex items-center justify-center
+                              align-middle rounded-full border-r-2  
+                              border-l-2 border-gray-400
+                              w-5 h-5"><img src="/images/infoIcon.svg" class="pt-0.5 h-5 inline"> </div>
+                </span>
+                <span class="tooltiptext text-xs">
+                    <h4 class="font-semibold">{{propertyItem.label}}</h4>
+                    <div v-html="propertyItem.description"></div>
+                </span>
+                </div>
+                <!-- Mobile Devices -->
+                <div v-else class="tooltip relative" >
```

Comment:
Not sure where to add this so just putting it here. When viewing on a mobile resolution, if you open a tooltip it seems to break the category grey box at the top. See the screenshot here:

![image](https://github.com/uoe-dlam/witches/assets/1457058/99ced521-e43d-476f-902c-123a31248d51)

Reply:
Just checked on tooltip and it's not the tooltip that's breaking this it is happening anyway for the longer titles e.g Social Classification and Wikipedia page, will try fix this on another branch.

Reply:
Fixed this in this branch "fixing-showing-icons-for-on-mobile" #116 .

Reply:
awesome, thanks Ruby